src/components/molecules/form/form-context.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+const LayoutContext = React.createContext()
```

Comment:
If we only have a single context for forms, won't we have problems if a user puts two forms with different layouts on the same page?

Reply:
Ah, I missed this part in the 16.3 docs:

> When React renders a context Consumer, it will read the current context value from the closest matching Provider above it in the tree.

Based on this, we should be fine, right? 😄